docs/src/topics/logging.md

```
@@ -306,6 +306,24 @@ LOGGING = {
 
 If no `LOGGING` is configured, Django-Bolt sets up sensible defaults with queue-based handlers.
 
+### Access log
+
+The Rust server writes one line per response to the `django.server` logger:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)